src/peergos/server/Start.java

```diff
@@ -25,8 +25,8 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-public class Start
-{
+public class Start {
```

Comment:
This whole class is based around shell commands that don't want ugly logging wrapping what they print to the user.